app/scripts/background.js

```diff
@@ -115,6 +115,7 @@ log.setLevel(process.env.METAMASK_DEBUG ? 'debug' : 'info', false);
 
 const platform = new ExtensionPlatform();
 const notificationManager = new NotificationManager();
+const isFirefox = getPlatform() === PLATFORM_FIREFOX;
```

Comment:
since we use `getPlatform() === PLATFORM_FIREFOX` in two other places in this same file, do you think this PR a good place to update those usages to this new top-level check?